cli/cmd/alert_list.go

```diff
@@ -51,8 +53,33 @@ Start and end times may be specified in one of the following formats:
     A. A relative time specifier
     B. RFC3339 date and time
     C. Epoch time in milliseconds
+
+To list open "NewViolations" alerts of high or critical severity.
```

Comment:
nit: I think this can be worded a bit better
```suggestion
To list open alerts of type "NewViolations" with high or critical severity.
```

Reply:
Sure. np.